firstvoices/backend/models/page.py

```diff
@@ -0,0 +1,65 @@
+from django.core.validators import validate_slug
+from django.db import models
+from django.db.models import Q
+from django.utils.translation import gettext as _
+
+from backend.models import Image
+from backend.models.base import BaseControlledSiteContentModel
+from backend.models.media import Video
+from backend.models.widget import SiteWidgetList
+from backend.permissions import predicates
+
+
+class SitePage(BaseControlledSiteContentModel):
+    class Meta:
+        verbose_name = _("site page")
+        verbose_name_plural = _("site pages")
+        rules_permissions = {
+            "view": predicates.is_visible_object,
```

Comment:
I would love to hear what the team thinks about the permissions for a page here. Currently, in this PR a page has its own visibility controls. Should a page's visibility be tied to the site visibility (if the site is visible then the page is visible) or is it helpful to have separate access control like this? @gmcauliffe @sarahfirstvoices 

Note: widgets have their own visibility controls already, so even if a page is visible that doesn't mean that all widgets associated with the page will be as well.

Reply:
That's great. Pages should have their own visibility. Thanks for checking 👍